docs/content/Caching/Lambda-Pre-Aggregations.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Lambda Pre-Aggregations
+permalink: /caching/pre-aggregations/lambda-pre-aggregations
+category: Caching
+menuOrder: 4
+---
+
+Lambda pre-aggregations follow the [Lambda architecture](https://en.wikipedia.org/wiki/Lambda_architecture) design to union real-time and batch data. Cube acts as a serving layer and uses pre-aggregations as a batch layer and source data or other pre-aggregations, usually [streaming][streaming-pre-agg], as a speed layer.
```

Comment:
Would it be better to say "service layer" (vs. "serving layer")?

Reply:
It is specific term used in the lambda architecture - https://en.wikipedia.org/wiki/Lambda_architecture